### PR TITLE
refactor: fully implementend log tree model

### PR DESCRIPTION
**Pull request by mvegter**

### CODE REVIEW

File: lib/public/Model.js [hunk 1]
```diff
@@ -59,6 +59,8 @@ export default class Model extends Observable {
                 this.logs.fetchAllLogs();
                 break;
             case 'entry':
+                this.logs.data = [];
+                this.logs.filteredData = [];
```
Comment (john-ghatas) [inline lib/public/Model.js:63]: Why initiate the arrays in the route switch case?

File: lib/public/components/Post/index.js [hunk 2]
```diff
@@ -12,19 +12,63 @@
...
+    id: {
+        name: 'ID',
+        visible: true,
+    },
+    authorId: {
+        name: 'Source',
+        visible: true,
+    },
+    creationTime: {
+        name: 'Created',
```
Comment (john-ghatas) [inline lib/public/components/Post/index.js:25]: Please verify that the datatype test (frontend tests) in the public folder still works

File: lib/public/Model.js (hunk 1 above)
Comment (mvegter) [reply lib/public/Model.js]: Some old code, has been removed.

File: lib/public/components/Post/index.js (hunk 2 above)
Comment (mvegter) [reply lib/public/components/Post/index.js:25]: The test for the datatype tests still works (as can be seen by GH actions), if that is what you meant.

File: lib/public/components/Post/index.js [hunk 3]
```diff
@@ -12,19 +12,63 @@
  */
 import { h } from '/js/src/index.js';
 
+const ACTIVE_FIELDS = {
```
Comment (john-ghatas) [inline lib/public/components/Post/index.js:15]: Any plans to unify this constant and save it in the ActiveColumns/index.js or any other location? Or does this variable server another purpose

**Comment by john-ghatas** [comment]
When resizing to a narrower screen the text overflows in the other postbox with the text contents like so:
image

File: lib/public/components/Post/index.js (hunk 3 above)
Comment (mvegter) [reply lib/public/components/Post/index.js:15]: I would say that each screen can have it own user preferences, so no this should not be merged.

**Comment by mvegter** [comment]
> When resizing to a narrower screen the text overflows in the other portion of the postbox with the text contents like so:
> image

This applies to all our screens, unless it is an issue on a 'normal' desktop resolution it is not a priority,

**Pull request merged**